src/App.js

```diff
@@ -17,7 +20,7 @@ function App() {
             <Home />
           </Route>
           <Route path="/profile">
-            <Profile />
+            {isAuth ? <Profile /> : <Redirect to="/" />}
           </Route>
           <Route exact path="/signin">
```

Comment:
Wellicht wil je ook niet dat een gebruiker die ingelogd is alsnog bij deze pagina kan. Hetzelfde geldt voor de signup pagina